<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>468</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>912</width>
    <height>468</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>912</width>
    <height>468</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Password Generator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QWidget {
	background-color: rgb(31, 31, 31);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>971</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>300</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>50</y>
            <width>471</width>
            <height>51</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>10000</width>
            <height>80</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
    color: rgb(255, 255, 255);
	letter-spacing: 2px
}</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Generate Password Which You Want&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>100</y>
            <width>381</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel { 
	color: rgb(159, 159, 159);
	letter-spacing: 2px;
}</string>
          </property>
          <property name="text">
           <string>The one and only Password Generator &lt;br&gt; which will help you to be more secure &lt;br&gt; Beacause Your PRIVACY is our PRIORITY</string>
          </property>
         </widget>
         <widget class="QFrame" name="frame_4">
          <property name="geometry">
           <rect>
            <x>640</x>
            <y>-10</y>
            <width>131</width>
            <height>131</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame { 
	background-color: rgb(255, 35, 15);
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="frame_6">
          <property name="geometry">
           <rect>
            <x>770</x>
            <y>120</y>
            <width>131</width>
            <height>131</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QFrame { 
	background-color: rgb(55, 202, 255);
}</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QSlider" name="length_slider">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>220</y>
            <width>207</width>
            <height>22</height>
           </rect>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change the length of the password by just sliding the slider&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QSlider::groove:horizontal {
    border: none;
    height: 3px; 
    background:rgb(81, 81, 81);
    margin: 2px 0;
}
QSlider::handle:horizontal {
    background: rgb(47, 182, 255);
    width: 10px;	
	margin: -7px 0; 
    border-radius: 5px;
}
QSlider::handle:horizontal:hover{
    background-color: rgb(34, 111, 255)
}</string>
          </property>
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>40</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>190</y>
            <width>251</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: rgb(175, 175, 175);
	letter-spacing: 1px;
}</string>
          </property>
          <property name="text">
           <string>Slide the length of the password</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="include_numbers">
          <property name="geometry">
           <rect>
            <x>340</x>
            <y>220</y>
            <width>171</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Check if you want to include numbers in your password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QCheckBox {
	color: rgb(175, 175, 175);
	letter-spacing: 1px;
}
QCheckBox::indicator {
    width: 15px;
    height: 15px;
	border-radius: 5px;
}

QCheckBox::indicator:unchecked {
   
	 background: rgb(47, 182, 255);
}
QCheckBox::indicator:unchecked:hover {
  
	background-color: rgb(34, 111, 255);
}
QCheckBox::indicator:checked {
    
	image: url(./assets/svg/check.svg);
	background: rgb(47, 182, 255);
}

QCheckBox::indicator:checked:hover {
    
	background-color: rgb(34, 111, 255);
}
</string>
          </property>
          <property name="text">
           <string>Include Numbers</string>
          </property>
         </widget>
         <widget class="QLabel" name="slider_value_label">
          <property name="geometry">
           <rect>
            <x>260</x>
            <y>220</y>
            <width>31</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: white;
}</string>
          </property>
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
	color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="generate_button">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>207</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This is a button to generate &lt;span style=&quot; color:#000000;&quot;&gt;password a&lt;/span&gt;ccording to your choice&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(79, 158, 255);
	border: none;
	color: white;
	letter-spacing: 1px;
	border-radius: 5px;
}
QPushButton:hover {
	background-color: rgb(34, 111, 255);
}
QPushButton::pressed {
	background-color: rgb(34, 111, 255);
}</string>
          </property>
          <property name="text">
           <string>Generate Password</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>80</y>
            <width>201</width>
            <height>16</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
	color: rgb(175, 175, 175);
	letter-spacing: 1px;
}</string>
          </property>
          <property name="text">
           <string>Generated Password :</string>
          </property>
         </widget>
         <widget class="QPushButton" name="exit_button">
          <property name="geometry">
           <rect>
            <x>300</x>
            <y>20</y>
            <width>171</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;This button will help to &lt;span style=&quot; font-weight:600;&quot;&gt;exit&lt;/span&gt; the Application&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
	background-color: rgb(239, 49, 1);
	border: none;
	color: white;
	letter-spacing: 1px;
	border-radius: 5px;
}
QPushButton:hover {
	background-color:rgb(209, 0, 0);
}
QPushButton::pressed {
	background-color: rgb(209, 0, 0);
}</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="output_edit">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>10</y>
            <width>431</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>ForbiddenCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;The generated password will be shown here&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
	background-color: rgb(74, 74, 74);
	border: none;
	color: white;
	border-radius: 10px;
	letter-spacing: 1px;
	padding: 5px;
}
QLineEdit::focus {
	border-bottom: 1px solid white;
}</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>The Generated Password will be shown here</string>
          </property>
         </widget>
         <widget class="QLabel" name="status">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>65</y>
            <width>271</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Montserrat</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Shows the status of the application&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#status {
	color: rgb(17, 255, 0);
	letter-spacing: 1.3px;
}
QLabel#error {
	color: red;
	letter-spacing: 1.3px;
}
</string>
          </property>
          <property name="text">
           <string>App Loaded Successfully ....</string>
          </property>
         </widget>
         <widget class="QPushButton" name="copy_button">
          <property name="geometry">
           <rect>
            <x>480</x>
            <y>11</y>
            <width>41</width>
            <height>41</height>
           </rect>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Copy to Clipboard&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../assets/svg/clipboard.svg</normaloff>../assets/svg/clipboard.svg</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>24</height>
           </size>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
